packages/web-extension/test/walletManager/WalletRepository.test.ts

```diff
@@ -13,7 +13,7 @@ import {
 } from '../../src';
 import { Cardano, Serialization } from '@cardano-sdk/core';
 import { Hash28ByteBase16 } from '@cardano-sdk/crypto';
-import { KeyRole } from '@cardano-sdk/key-management';
+import { KeyPurpose, KeyRole } from '@cardano-sdk/key-management';
```

Comment:
Need to add some WalletRepository tests:
```
it('allows creating 1852 and 1854 purpose account with the same index');
describe('repository has 1852 and 1854 purpose accounts that have the same accountIndex', () => {
  it('does not remove 1852 account when removing 1854 account );
  it('does not update 1852 account metadata when updating 1854 account');
});
```

Reply:
Add tests on 509bdd9